src/components/Button.tsx

```diff
@@ -48,12 +48,18 @@ export const DropDownButton: React.SFC<DropDownButtonProps> = ({
 }) => {
   const _className = cx(
     css({
-      border: constants.border.base
+      border: constants.border.base,
+      "&:not([disabled]):hover": {
+        backgroundColor: colors.gray.lighter
+      }
```

Comment:
please move styles to styles file. even if it was present here earlier, you should do what you think is right. :)